app/src/main/assets/GimelGimel/VectorLayer.js

```diff
@@ -185,7 +192,28 @@ function symbolToImageMarkerSymbol(symbol) {
         height: symbol.imageHeight,
         verticalOrigin: Cesium.VerticalOrigin.BOTTOM
     }
-}
+};
+
+var addAltitudeToLocation = function (location) {
+    var promise = new Promise(
+        function (resolve, reject) {
+            if (!location.hasAltitude) {
+                var point = Cesium.Cartographic.fromDegrees(location.longitude, location.latitude, 0, new Cesium.Cartographic());
+
+                Cesium.sampleTerrain(GG.viewer.terrainProvider, 9, [point])
```

Comment:
Extract the terrain resolution sampling parameter into a constant with a meaningful name.